pages/common/hub.md

```diff
@@ -20,6 +20,10 @@
 
 `hub pull-request --no-edit`
 
+- Create a PR against a specified branch instead of the default branch:
+
+`hub pull-request --base {{dev}}`
```

Comment:
I think we should use the same placeholder as `gh-pr-create`:
```suggestion
`hub pull-request --base {{base_branch}}`
```

Reply:
Sure, we can do that as well, but since we do not have a page for it ? Should we probably add one `hub-pull-request` ?

Reply:
Yeah, add it.